java/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM eclipse-temurin:17.0.6_10-jdk
+
+WORKDIR /lib
+
+# Install protoc (cf. http://google.github.io/proto-lens/installing-protoc.html)
+ENV PROTOC_ZIP=protoc-21.12-linux-x86_64.zip
+RUN apt-get update && apt-get install -y unzip
+RUN curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v21.12/$PROTOC_ZIP \
+    && unzip -o $PROTOC_ZIP -d /usr/local bin/protoc \
+    && unzip -o $PROTOC_ZIP -d /usr/local 'include/*' \
+    && rm -f $PROTOC_ZIP
+
+# Install maven
+ENV MAVEN_VERSION=3.9.0
+RUN curl -OL https://dlcdn.apache.org/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.zip \
+    && unzip -o apache-maven-$MAVEN_VERSION-bin.zip -d /usr/local \
+    && rm -f apache-maven-$MAVEN_VERSION-bin.zip
+ENV PATH=$PATH:/usr/local/apache-maven-$MAVEN_VERSION/bin
```

Comment:
Can you use the official Maven docker images instead?  https://hub.docker.com/_/maven

Reply:
The official Maven docker image works!